packages/iota/src/clients/singleNodeClient.ts

```diff
@@ -122,8 +117,7 @@ export class SingleNodeClient implements IClient {
             throw new Error("The endpoint can not be empty");
         }
         this._endpoint = endpoint.replace(/\/+$/, "");
-        this._basePath = options?.basePath ?? "/api/v2/";
-        this._basePluginPath = options?.basePluginPath ?? "/api/plugins/";
```

Comment:
Maybe we can change this line to  `this._baseApiPath = this._basePath + "core/v2/"; `
And than use this instead of this.coreApiPath()